apps/condo/domains/contact/components/ContactsEditor/index.tsx

```diff
@@ -71,7 +71,7 @@ export const ContactsEditor: React.FC<IContactEditorProps> = (props) => {
     searchContacts(client, {
         organizationId: organization,
         propertyId: property ? property : undefined,
-        unitName: unitName ? unitName : undefined,
+        unitName: unitName ? unitName : null,
```

Comment:
This line was already changed in PR #774
I've commented about different solution for future.